src/css/_details.scss

```
@@ -0,0 +1,43 @@
+@use "./functions" as *;
+
+// Docusaurus hardcodes `alert alert--info` onto every <details>. details.alert
+// is (0,1,1) and outranks .alert--info without touching :::info admonitions.
+details.alert {
+  --details-caret-size: #{rem(16px)};
+  --details-border-color: var(--ifm-color-emphasis-300);
+  // Inverts per theme. Caret only; the divider follows the border.
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)